test/framework/shoot_utils.go

```diff
@@ -160,13 +160,50 @@ func CreateShootTestArtifacts(cfg *ShootCreationConfig, projectNamespace string,
 		return "", nil, err
 	}
 
+	if err := setShootAnnotations(shoot, cfg); err != nil {
```

Comment:
This would be a great fit for `setShootMetadata`, right? Do you want to call it from there?

Reply:
makes sense